condition/scenario_conditions/package.xml

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0"?>
-
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>scenario_conditions</name>
   <version>0.0.0</version>
   <description>The scenario_conditions package</description>
-
   <maintainer email="ms.kataoka@gmail.com">Masaya Kataoka</maintainer>
 
-  <license>TODO</license>
+  <license>Apache License 2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
+  <depend>rclcpp</depend>
   <depend>pluginlib</depend>
-  <depend>roscpp</depend>
   <depend>scenario_api</depend>
   <depend>scenario_intersection</depend>
   <depend>scenario_utility</depend>
   <depend>visualization_msgs</depend>
-  <depend>yaml-cpp</depend>
+  <depend>yaml_cpp_vendor</depend>
```

Comment:
I forgot to add the boost dependency here while I was doing the original port

Reply:
Done!

Reply:
but doesn't that require `target_link_libraries()` call in `CMakeLists.txt` to link to boost in a correct manner, if boost istn' in the system paths the linker uses?

Reply:
I simply removed the Boost dependency. See the stringstream above.